rever.xsh

```diff
@@ -1,10 +1,15 @@
 $REVER_DIR = 'rever-tmp'
-$ACTIVITIES = ['version_bump', 'changelog', 'tag']
+$ACTIVITIES = ['version_bump', 'changelog', 'tag', 'conda_forge']
 
 $VERSION_BUMP_PATTERNS = [
     ('rever/__init__.py', '__version__\s*=.*', "__version__ = '$VERSION'"),
     ('setup.py', '        version\s*=.*,', "        version='$VERSION',")
     ]
 $CHANGELOG_FILENAME = 'CHANGELOG.rst'
 $CHANGELOG_IGNORE = ['TEMPLATE.rst']
-$TAG_REMOTE = 'git@github.com:ergs/rever.git'
+$TAG_REMOTE = 'git@github.com:regro/rever.git'
+$GITHUB_ORG = 'regro'
+$GITHUB_REPO = 'rever'
+$PROJECT = 'rever'
+$GITHUB_CREDFILE = '~/.gh_cred.txt'
```

Comment:
This should not be set in the rever file ever.  This is for user's to set in their environment, if needed.  There will one credential file per project so we especially don't want to set this to something that would be shared by projects.

Reply:
It would be nice if we could figure out a way to warn or prevent...

Reply:
Actually, I think I know how...

Reply:
Ah ok, I wasn't certain.